app/templates/releases/lts.hbs

```diff
@@ -18,10 +18,14 @@
     since LTS versions get even more testing and scrutiny than usual.
   </p>
   <p>
-    An LTS is declared roughly every four minor versions, plus the final version before a
-    major release. Before a version can be called an "LTS" release, it has to spend
-    at least 6 weeks as a stable release, where it is used and tested by thousands of
-    developers.
+    An LTS is declared roughly every 4 minor versions, excluding the x.0 minor version.
```

Comment:
Is their ambiguity here still? I'm not sure if excluding implies that we start counting at x.0 or x.1

x.0
x.1
x.2
x.3 <-- fourth version
x.4 <-- actual LTS

I can be convinced the above isn't ambiguous, but i wonder if something like. Starting after <clause> helps? Ultimately this is still an improvement.

Reply:
Hmm. I think we may assume that a developer who visits the LTS page is somewhat familiar with semantic versioning. In particular, the developer understands that the major, minor, and patch version numbers begin at 0. (At least, this is my understanding of version numbers.)

So, every 4 minor versions would mean `0, 4, 8, ...`. Afterwards, excluding 0 would result in `4, 8, ...`.

In a subsequent sentence, I provided an example with Ember `2.x` series, in case the pattern may not be still clear to a developer. As a result, I think what's currently written would be okay. What are your thoughts?